hafnian/samples.py

```diff
@@ -458,3 +466,53 @@ def torontonian_sample_graph(A, n_mean, samples=1, max_photons=30, pool=False):
     Q = gen_Qmat_from_graph(A, n_mean)
     cov = Covmat(Q)
     return torontonian_sample_state(cov, samples, hbar=2, max_photons=max_photons, pool=pool)
+
+
+def hafnian_sample_classical_state(cov, samples, mean=None, hbar=2, sigdigits=6):
+    r""" Returns samples from a Gaussian state that has a positive P function
+    Args:
+        cov(array): a :math:`2N\times 2N` ``np.float64`` covariance matrix
+            representing an :math:`N` mode quantum state. This can be obtained
+            via the ``scovmavxp`` method of the Gaussian backend of Strawberry Fields.
+        samples (int): number of samples to generate
+        mean (array): (default zero) vector of means of the gaussian state
+        hbar (float): (default 2) the value of :math:`\hbar` in the commutation
```

Comment:
I don't think we need to include defaults, they should already be part of the docstring in sphinx:
![image](https://user-images.githubusercontent.com/2959003/59891051-afcd5d00-93a1-11e9-8b07-3f7ffb00a9c5.png)

Reply:
Thoughts on this comment?